packages/quantic/force-app/main/default/lwc/quanticRecentQueries/quanticRecentQueries.js

```diff
@@ -0,0 +1,83 @@
+import { LightningElement, api, track } from 'lwc';
+import { registerComponentForInit, initializeWithHeadless } from 'c/quanticHeadlessLoader';
+import { getItemFromLocalStorage, setItemInLocalStorage } from 'c/quanticUtils';
+import recentQueriesLabel from '@salesforce/label/c.quantic_RecentQueries';
+import emptyListLabel from '@salesforce/label/c.quantic_EmptyRecentQueriesLabel';
+
+export default class QuanticRecentQueries extends LightningElement {
+  labels = {
+    recentQueriesLabel,
+    emptyListLabel,
+  }
+  
+  /** @type {import("coveo").RecentQueriesState} */
+  @track state;
+  /** @type {string} */
+  @api label = this.labels.recentQueriesLabel;
+  /** @type {string} */
+  @api engineId;
+  /** @type {import("coveo").RecentQueriesList} */
+  recentQueriesList;
+  /** @type {()=> void} */
+  unsubscribe;
+  /** @type {boolean} */
+  isCollapsed = false;
+  /** @type {string} */
+  collapseIcon = 'utility:dash';
+  /** @type {number} */
+  @api maxLength = 10;
+
+  connectedCallback() {
+    registerComponentForInit(this, this.engineId);
+  }
+
+  renderedCallback() {
+    initializeWithHeadless(this, this.engineId, this.initialize.bind(this));
+  }
+
+  /**
+  * @param {import("coveo").SearchEngine} engine
+  */
+  @api
+  initialize(engine) {
+    this.recentQueriesList = CoveoHeadless.buildRecentQueriesList(engine, {
+      initialState: {
+        queries: getItemFromLocalStorage(this.localStorageKey) ?? [],
+      },
+      options: {
+        maxLength: this.maxLength,
```

Comment:
```suggestion
maxLength: Number(this.maxLength)
```
I made the mistake the first time